cmd/Makefile.am

```diff
@@ -5,4 +5,4 @@ if USING_PYTHON
 SUBDIRS += arcstat arc_summary dbufstat
 endif
 
-SUBDIRS += mount_zfs zed zvol_id
+SUBDIRS += mount_zfs zed zvol_id zfs_volume_wait
```

Comment:
To resolve this build error you'll need to add the new utility to the spec file.

```
Checking for unpackaged file(s): /usr/lib/rpm/check-files /tmp/zfs-build-buildbot-XOz43uTI/BUILDROOT/zfs-0.8.0-106_g663bba6.el7.x86_64
error: Installed (but unpackaged) file(s) found:
   /usr/bin/zfs_volume_wait
```

This should work.

```diff
index 0b16cd0..989a31b 100644
--- a/rpm/generic/zfs.spec.in
+++ b/rpm/generic/zfs.spec.in
@@ -418,6 +418,7 @@ systemctl --system daemon-reload >/dev/null || true
 %{_sbindir}/*
 %{_bindir}/raidz_test
 %{_bindir}/zgenhostid
+%{_bindir}/zfs_volume_wait
 # Optional Python 2/3 scripts
 %{_bindir}/arc_summary
 %{_bindir}/arcstat
```

Would it be more convenient and consistent to call the command `zvol_wait` similar to `zvol_id`?  Or do we want the longer name, I don't have a preference but wanted to ask.

Reply:
> Would it be more convenient and consistent to call the command zvol_wait similar to zvol_id? Or do we want the longer name, I don't have a preference but wanted to ask.

I'm fine either way. I've called it zfs_volume_wait because the service was renamed to `zfs-volume.service`.

Reply:
renamed the script to `zvol_wait`